### PR TITLE
[containers-int-strict]: Inline and fexpose-all-unfoldings

### PR DESCRIPTION
**Pull request by piq9117**
added INLINE pragma, and added fexpose-all-unfoldings flag to cabal file.

Is it okay to submit multiple tiny PRs for https://github.com/kowainik/containers-backpack/issues/33? That way it will be easier to review?

### CODE REVIEW

**Comment by vrom911** [comment]
Don't be shy to ask questions about `git` as well if you have any issues with that! It's okay to mess with it (I've done it as well) and it's nice to learn on that 🙂

File: containers-ordered-strict/src/Map/Ord.hs [hunk 1]
```diff
@@ -35,12 +35,16 @@ type Key = Ord
 
 lookupDefault :: Key k => v -> k -> Map k v -> v
 lookupDefault = M.findWithDefault
+{-# INLINE lookupDefault #-}
```
Comment (ChShersh) [inline containers-ordered-strict/src/Map/Ord.hs]: I think we should add `-fexpose-all-unfoldings` to the `containers-ordered-strict` package as well.

File: containers-primitive/src/Map/Prim.hs [hunk 2]
```diff
@@ -44,6 +45,7 @@ lookupDefault def k = fromMaybe def . M.lookup k
 
 toList :: Key k => Map k v -> [(k, v)]
 toList = L.toList
+{-# INLINE toList #-}
 
 keys :: Key k => Map k v -> [k]
 keys = map fst . toList
```
Comment (ChShersh) [inline containers-primitive/src/Map/Prim.hs]: Some functions in this module miss `{-# INLINE #-}` pragma.

File: containers-unordered-strict/containers-unordered-strict.cabal [hunk 3]
```diff
@@ -28,6 +28,5 @@ library
   build-depends:       base
                      , hashable
                      , unordered-containers
-
   ghc-options:         -Wall -fhide-source-paths
```
Comment (ChShersh) [inline containers-unordered-strict/containers-unordered-strict.cabal]: Probably here we also should add `-fexpose-all-unfoldings`

**Comment by piq9117** [comment]
Thanks for taking the time to review my PR. i'll be making those changes.

**Comment by ChShersh** [comment]
@piq9117 Thanks to you for working on this PR!

**Pull request merged**